internal/resource/ca_certificate_test.go

```diff
@@ -76,7 +76,7 @@ p0TGlNicxb2nnhRuh1U63VkBhl3k17vClDiwqHYwe7fPzlmj6kZMaXnpK6rd4kg7
 -----END CERTIFICATE-----
 `
 
-	invalidCert = `
+	invalidCertOne = `
```

Comment:
nit: Should we throw some comments on these certs on how they were created?

Reply:
It'd be a good idea, if I had notes 🤦

Reply:
Yeah I was mostly concerned what it is, allows CA, and whatnot.